addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -739,13 +761,56 @@ QUnit.test('use a command', async function (assert) {
     );
 });
 
+QUnit.test("channel with no commands should not prompt any command suggestions on typing /", async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({ channel_type: 'chat', id: 20 });
+    this.data['mail.channel_command'].records.push(
+        {
+            channel_types: ['livechat'],
+            help: "See 15 last visited pages",
+            name: "history",
```

Comment:
Prefer a command whose behaviour actually exists, like `/who` or `/leave`.
Channel commands are in crisis by being partially data and partially server inherent behaviour.
Actual python use pure server inherent behaviour, with illusion these are actual data when passed to client via `init_messaging`. Should have been data in 1st place, but can't do that in one night...

Reply:
Done